### PR TITLE
Add a couple of spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -6781,6 +6781,8 @@ reacing->reaching
 reacll->recall
 readabilty->readability
 readapted->re-adapted
+readded->read
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: http://www.yourdictionary.com/readd

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: hah, didn't read the word that way. Nice catch. How about:
```
readded->readded, read,
readed->read, readd, readded,
```

Comment (larsoner) [reply codespell_lib/data/dictionary.txt]: sounds good

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: ok.. revised an push -f 'ed

**Pull request merged**